### PR TITLE
Quasar framework v0.17

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

**Review by mattcollier — CHANGES_REQUESTED** [review]
The release script is responsible for updating the version in the package file.  Unfortunately changing it like this will break the release script.  So, please back that part out, and whoever releases it will need to make sure to do a major release.  You could indicate this by updating the changelog, and leave the release date TBD to be filled out by the person that does the release.

**Comment by gannan08** [comment]
Reverted the manual version change.

File: package.json [hunk 1]
```diff
@@ -24,13 +24,14 @@
...
     "firefox-iframe-getcomputedstyle": "^1.0.0",
-    "quasar-framework": "^0.15.10"
+    "quasar-framework": "^0.17.14"
   },
   "bedrock": {
     "browserDependencies": "all",
     "manifest": {
       "quasar-framework": {
         "main": "dist/umd/quasar.mat.umd.js",
+        "module": "dist/umd/quasar.mat.umd.js",
```
Comment (dlongley) [inline package.json:34]: Is adding "module" here the right thing to do? We didn't need it before ... what changed?

Comment (gannan08) [reply package.json:34]: Not doing so makes bedrock-views pull in the esm version of quasar causing it to break during development mode. What changed was the addition of "module" in quasar's package.json https://github.com/quasarframework/quasar/commit/0ac2e638f5ac47d522aaa37205fd81db86c6b6b1

Comment (dlongley) [reply package.json:34]: I see. Good to know. Did you check to see what happens when we optimize now? Does it include multiple versions of the same lib?

Comment (gannan08) [reply package.json:34]: It does not.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**